homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -835,7 +837,12 @@ def __init__(self, device, entry, unique_id, coordinator):
             self._device_features = FEATURE_FLAGS_FAN_1C
         elif self._model == MODEL_FAN_P9:
             self._device_features = FEATURE_FLAGS_FAN_P9
-        elif self._model in (MODEL_FAN_P10, MODEL_FAN_P11):
+        elif self._model in (
+            MODEL_FAN_P10,
+            MODEL_FAN_P11,
+            MODEL_FAN_P15,
+            MODEL_FAN_P18,
+        ):
             self._device_features = FEATURE_FLAGS_FAN_P10_P11
```

Comment:
IMO the name of this const `FEATURE_FLAGS_FAN_P10_P11` should be changed to `FEATURE_FLAGS_FAN_P10_P11_P15_P18`.

Reply:
Done 👍